### PR TITLE
Update Ivan.html

### PR DESCRIPTION
**Pull request by IvanNik85**

### CODE REVIEW

File: Ivan.html [hunk 1]
```diff
@@ -1 +1,26 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>page2</title>
+</head>
+<body>
+    <h1>Druga strana: idemoooooo!!!</h1>
+    <p>
```
Comment (Jolene986) [inline Ivan.html:12]: NE MOZE TO TAKO!!!!

Comment (IvanNik85) [reply Ivan.html:12]: Porke No?!

Comment (Jolene986) [reply Ivan.html:12]: his branch has conflicts that must be resolved

**Review by Jolene986 — CHANGES_REQUESTED** [review]

File: Ivan.html (hunk 1 above)
Comment (IvanNik85) [reply Ivan.html:12]: AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA!!!!!

Comment (IvanNik85) [reply Ivan.html:12]: 💃

**Pull request closed without merging**